tests/virt_autotest/set_config_as_glue.pm

```diff
@@ -31,7 +31,11 @@ sub fufill_guests_in_setting {
     my $get_vm_hostnames   = "virsh list  --all | grep sles | awk \'{print \$2}\'";
     my $vm_hostnames       = script_output($get_vm_hostnames, $wait_script, type_command => 0, proceed_on_failure => 0);
     my @vm_hostnames_array = split(/\n+/, $vm_hostnames);
-    foreach (@vm_hostnames_array) { $xen::guests{$_} = '1'; }
+    foreach (@vm_hostnames_array) {
+        my $get_vm_macaddress = "virsh domiflist --domain $_ | grep -oE \"([0-9|a-z]{2}:){5}[0-9|a-z]{2}\"";
```

Comment:
This is totally nonsense. The MAC address is already defined in `$xen::guests{$_}->{macaddress}` which you even use so why you try to parse it again?

Reply:
Please differentiate QA-SLE and QAM code when handling xen::guests parameter. We are different here.

Reply:
Same to the above "We have different vm names and so mac addresses. Although we reuse variables and test functions from xen.pm, we still need to override it to load our vm info in our tests."

Reply:
When we in future reuse the virt-manager test code, to reuse needles, we may add glue script to rename our guest names to theirs. Please handle differently as my above comment suggested.

Reply:
I just noticed, this glue file will not be executed by QAM tests, so you can ignore my comment here.